src/pages/index.tsx

```diff
@@ -1,17 +1,24 @@
+import React from 'react'
 import type { NextPage } from 'next'
 import Head from 'next/head'
-import React from 'react'
+import dynamic from 'next/dynamic'
+
+// Components using document or window elements must deactivate ssr to be used on client side
+// https://nextjs.org/docs/advanced-features/dynamic-import#with-no-ssr
+const ContentWithoutSSR = dynamic(async () => import('../components/content/Content'), {
+  ssr: false
+})
 
 const Home: NextPage = () => {
   return (
-    <div>
+    <div className="okp4-faucet-testnet-main">
       <Head>
-        <title>Create Next App</title>
+        <title>OKP4 Faucet Testnet</title>
         <meta content="Generated by create next app" name="description" />
```

Comment:
is this `meta` tag useful? I assume it's a generic tag left behind from the `create next app` template?